tests/test-cross-context-dispatch/ccd-common/src/main/java/org/eclipse/jetty/tests/ccd/common/Property.java

```diff
@@ -0,0 +1,59 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.tests.ccd.common;
```

Comment:
Can this class be a `record` instead?

Reply:
our json lib didn't like that

Reply:
Must say something about the age of the json library :)